### PR TITLE
Upgrade of Profile_Activity Layout + image in search user activity

### PR DESCRIPTION
**Pull request by MaximeZmt**
photo_2022-05-16_09-45-04
photo_2022-05-16_13-27-44

Upgrade of Profile_Activity Layout: Fit better on any smartphone now
Update images loading to new version for Search User Activity

### CODE REVIEW

**Review by kamilababayeva — COMMENTED** [review]
Thanks for your pr! I have some suggestions, let me know if you can address them. Also, one of the idea I changed to relative layout was that the tests were failing. They fail now as well.
- For some reason, in my phone the button LogOut is too close to score stats. I have attached the picture below.
- If the username is too long, the button to edit and GQ code disappears from the screen
- Maybe we should do these button static and not move based on username length

image

**Review by Tsathogguaa — APPROVED** [review]
Good short fix, thank you!

File: app/src/main/java/ch/sdp/vibester/activity/ProfileActivity.kt [hunk 1]
```diff
@@ -89,6 +90,9 @@ class ProfileActivity : AppCompatActivity() {
         findViewById<ImageView>(R.id.avatar).setOnClickListener {
             showGeneralDialog( "Do you want to change your profile picture?", false)
         }
+        findViewById<CardView>(R.id.myCardView).setOnClickListener {
+            showGeneralDialog( "Do you want to change your profile picture?", false)
```
Comment (kamilababayeva) [inline app/src/main/java/ch/sdp/vibester/activity/ProfileActivity.kt:94]: can we test this?

File: app/src/main/java/ch/sdp/vibester/activity/ProfileActivity.kt [hunk 2]
```diff
@@ -159,7 +164,8 @@ class ProfileActivity : AppCompatActivity() {
     private fun setQrCodeToProfileBtnListener() {
         findViewById<FloatingActionButton>(R.id.qrCode_returnToProfile).setOnClickListener {
             setLayoutVisibility(findViewById<ConstraintLayout>(R.id.QrCodePage), false)
-            setLayoutVisibility(findViewById<RelativeLayout>(R.id.profileContent), true)
+            setLayoutVisibility(findViewById<ScrollView>(R.id.profileContent), true)
```
Comment (kamilababayeva) [inline app/src/main/java/ch/sdp/vibester/activity/ProfileActivity.kt:167]: can we test this?

**Review by kamilababayeva — APPROVED** [review]
Awesome, thanks!

**Pull request merged**